transcendence-app/seeds/seed_users.ts

```diff
@@ -0,0 +1,32 @@
+import { Knex } from 'knex';
+import { faker } from '@faker-js/faker';
+
+const USERS_NUMBER = 5000;
+
+const createRandomUser = () => {
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const username = faker.helpers.unique(faker.internet.userName, [
```

Comment:
Either we make the fullName unique, or we make the username editable. 
From the scale:
```
When logged in, the user has access to his admin panel where he can change informations about himself,
including its nickname (must be unique across the users of the website)
```

Reply:
Good point :+1:

We could make the username editable and save the id from the intranet to check if the user exists or not because currently, we use the username for this. We could add two more columns to the user table: `provider` and `providerId`. This way, we could also add more authentication options(github, google, etc.) in the future.